### DISCUSSION TITLE
How i can to override scss-variable $font-family?

### DISCUSSION THREAD

**Discussion by RagushinIA** [element-plus/element-plus#11582] [Q&A]
Hello!
I understand how to override variables $colors or $border-radius.
How i can to override variable $font-family?
I try to override $font-family in what way:

```
@forward 'element-plus/theme-chalk/src/common/var.scss' with (
    $font-family: (
        '':
        "'Tahoma', Helvetica, 'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', '微软雅黑', Arial, sans-serif"
    )
)
```

It doesn't work.

**Comment by btea** [comment]
This configuration should generate the corresponding css variable `--el-font-family`, you can set `font-family: var(--el-font-family)` where you need to use it.
